releasenotes/notes/add-support-to-disable-amplitude-limit-in-parametric-pulses-ef88b77db8c1b06c.yaml

```diff
@@ -0,0 +1,10 @@
+---
+features:
+  - |
+    Adds an argument 'limit_amplitude' to the constructor of ParametricPulse
```

Comment:
```suggestion
    Adds an argument ``limit_amplitude`` to the constructor of ParametricPulse
```

Reply:
Thanks @taalexander! I've made the changes.